lerna.json

```diff
@@ -1,8 +1,5 @@
 {
-  "packages": [
-    "packages/*"
-  ],
-  "version": "1.0.0",
+  "version": "independent",
```

Comment:
why this?

Reply:
[It lets you bump versions of your packages independent of each other.](https://github.com/lerna/lerna#independent-mode)